cmd/mircat/debug.go

```diff
@@ -19,6 +16,8 @@ import (
 	"github.com/filecoin-project/mir/pkg/pb/recordingpb"
 	t "github.com/filecoin-project/mir/pkg/types"
 	"google.golang.org/protobuf/encoding/protojson"
+	"io"
+	"os"
```

Comment:
Why moving these imports here?

Reply:
Hmm, my editor automatically separates built-in imports imports from normal ones since `gofmt -s` doesn't like the other way.